docs/reference/action_list.rst

```diff
@@ -244,6 +255,36 @@ Configuring the default ordering column can be achieved by overriding the
         // ...
     }
 
+Default ordering may be defined also in overwritten ``\Sonata\AdminBundle\Admin\AbstractAdmin::createQuery() method``
+directly on the query used to load data::
+
+    // src/Admin/PostAdmin.php
+
+    use Sonata\AdminBundle\Admin\AbstractAdmin;
+
+    final class PostAdmin extends AbstractAdmin
+    {
+        // ...
+
+        /**
+         * {@inheritdoc}
+         */
+        public function createQuery($context = 'list'): ProxyQuery
+        {
+            /** @var ProxyQuery $query */
+            $query = parent::createQuery($context);
+
+            return $query
+                ->setSortOrder('DESC')
+                ->setSortBy([], ['fieldName' => 'updatedAt'])
+                ;
+        }
+
+        // ...
+    }
+
+It may be useful when the sorting depends on custom logic that can't be applied to the ``datagridValues`` property.
```

Comment:
```suggestion
It may be useful when the sorting depends on custom logic that can't be applied to the ``$datagridValues`` property.
```

to be consistent with the first sentence